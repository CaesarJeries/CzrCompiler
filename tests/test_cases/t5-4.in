void main() {
//Arithmetic test

int a = 4;
int e = 3;

a = (a + e) * a + e - a * e / e * e;
e = (a + e) * (e - a) * (e + 3*a);
printi(a);
printi(e);

byte c = 12b;
int d = a+c/10-e*c;
printi(d);

}