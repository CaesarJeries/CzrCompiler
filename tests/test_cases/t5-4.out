Loaded: ./exceptions.s
19-21120253460